mage_ai/frontend/package.json

```diff
@@ -47,7 +52,7 @@
     "moment-timezone": "^0.5.43",
     "monaco-editor": "^0.44.0",
     "monaco-themes": "^0.4.4",
-    "next": "12.3.4",
+    "next": "14.0.3",
```

Comment:
@mfreeman451 Let's upgrade to next 14.0.4 (released 5 days ago). Looks like there were some opentelemetry fixes in the [changelog](https://github.com/vercel/next.js/releases/tag/v14.0.4)

Reply:
We should also upgrade to `react@18.2.0` and `react-dom@18.2.0` per the Next [docs](https://nextjs.org/docs/messages/react-version).

Reply:
bumped